app/uk/gov/hmrc/timetopayproxy/models/CreatePlanResponse.scala

```diff
@@ -62,6 +63,7 @@ object PlanStatus extends Enum[PlanStatus] with PlayJsonEnum[PlanStatus] {
       case "Resolved - Cancelled"          => ResolvedCancelled
       case "Resolved - Completed"          => ResolvedCompleted
       case "TTP - Monitoring suspended"    => MonitoringSuspended
+      case "Pending - Cancellation Letter" => PendingCancellationLetter
```

Comment:
I wonder if it may be better to match on `entryName` for all the cases like below in order to avoid typographic divergence between the strings.
```scala
case PendingCancellationLetter.entryName => PendingCancellationLetter
```
What do you guys think?

Reply:
Oh yeah, I think we should probably use the built in value of method. Just realised that this case statement isn't total as well, we should probably do something sensible in the case where none of them match as well

Reply:
I'll amend and create a ticket to do the same in the time-to-pay.